README.md

```diff
@@ -646,26 +646,19 @@ class PostVersion < PaperTrail::Version
 end
 ```
 
-If you only use custom version classes and don't use PaperTrail's built-in one,
-on Rails `>= 3.2` you must:
-
-- either declare the `PaperTrail::Version` class to be abstract like this (in an
-  initializer):
+If you only use custom version classes and don't have a `versions` table, you
+must let ActiveRecord know that `PaperTrail::Version` is an `abstract_class`.
```

Comment:
I kind of liked the phrasing of "know that the `PaperTrail::Version` class is an `abstract_class`", just to be explicit about saying up front that `PaperTrail::Version` is a class to begin with (even though I understand it's implied)